### PR TITLE
Update security-center-incident.md

### PR DESCRIPTION
**Pull request by yoavfrancis**
Typos + blade name fix (The blade name is not always "Security incident detected")

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@yoavfrancis__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
Hi __@yoavfrancis__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by PRmerger** [comment]
@yoavfrancis : Thanks for your contribution to the Azure documentation! The author, @YuriDio, has been notified to review your proposed change.

**Comment by YuriDio** [comment]
@yoavfrancis  thanks for your contribution!
Regarding your comments: 
2nd comment: security alerts tile < The tile that I'm referring to at this point is upper case. So please leave as is.

3rd comment: "shown in this image" instead of "below" as you suggested: for acessibility compliance reason, we are reviewing all articles to remove reference to "below" or "above". 

4th comment: "Security incident detected blade" < this is regarding the example that I'm referring to, please leave as is.

In summary, we will merge the first comment.

Again, thanks for your contribution.

@BryanLa see my observations and just merge the first suggestion.

**Comment by BryanLa** [comment]
@yoavfrancis @YuriDio - unfortunately merging is an all-or-nothing operation on the commits. Any way one of you can push another commit to back out all changes except the first one?

**Comment by YuriDio** [comment]
@BryanLa , in this case merge as is and I will make the appropriate changes later. Thanks!
